tests/e2e/lib/pages/wp-admin/plugins.js

```diff
@@ -40,16 +40,15 @@ export default class PluginsPage extends Page {
 	}
 
 	async updateJetpack() {
-		await this.page.waitFor( 2000 );
-		const updateCard = 'tr.active#jetpack-update[data-plugin="jetpack/jetpack.php"]';
```

Comment:
I guess it would make sense to revert _all_ these changes, as it was reverted in `getJetpackVersion`

Reply:
I guess I missed those. All should be reverted now.